.github/workflows/ci-n8n.yaml

```diff
@@ -0,0 +1,17 @@
+name: CI for n8n
+on:
+  workflow_call:
+jobs:
+  ci:
+    name: Run CI for n8n
+    runs-on: ubuntu-latest
```

Comment:
Not using our own runner becoz its public?

Reply:
As we are only running CI here and somehow self hosted runner get disconnected after turning into public, so didn't setup self hosted runner here.

cc @jasonkit